README.md

```diff
@@ -10,6 +10,16 @@ JIFBOX is a web-based animated GIF photobooth. GIFs are generated in the browser
 
 Want to see some GIFs generated by JIFBOX? Check out Sunlight Foundation's [Sunfright Tumblr](http://sunfrightgifs.tumblr.com) from our annual Halloween open house.
 
+## Install and Setup
+1. Clone this repo and `cd` into the directory.
+2. Create a virtual environment. (If you have [virtualenvwrapper](https://virtualenvwrapper.readthedocs.org) you can run `mkvirtualenv jifbox`)
+3. Run `pip install -r requirements.txt` to install dependencies.
+4. Run `cp .env.example .env` to make your own copy of the .env file.
+5. Run `source .env` to load environment variables.
+5. Make sure MongoDB is running. If you don't have MongoDB, install it using `brew install mongodb` or follow [these instructions to install and run MongoDB on your OS](https://docs.mongodb.org/manual/installation/#installation-guides).
```

Comment:
Also, we had issues running a local instance of mongo. I found it easier to set up a mongolab account.